test/common/http/filter/lua/lua_filter_test.cc

```diff
@@ -633,6 +635,7 @@ TEST_F(LuaHttpFilterTest, RequestAndResponse) {
 
   TestHeaderMapImpl request_headers{{":path", "/"}};
   EXPECT_CALL(*filter_, scriptLog(spdlog::level::trace, StrEq("/")));
+  EXPECT_CALL(decoder_callbacks_, clearRouteCache());
```

Comment:
Would a test to check that the reroute actually happens also be useful, or is it overkill?

Reply:
The only way to do that would be to add a much more complex integration test. I could do it, but probably overkill IMO.

Reply:
Sure, no worries, fine as is.